### DISCUSSION TITLE
Whitebeet User Manual V2.14

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#84] [Q&A]
In section 11.3.4 of the User Manual (Set Serial Number), an example is given for serial number: (Ser00001). The data bytes for this serial number is given as '08 53 65 72 30 30 30 30 31'. Is this correct? If so what does '08' represent? Cheers. Rob.

**Comment by jpo-stx** [comment]
Hello @Bobbybigears 
the example is correct. The serial number needs to be prefixed with one byte defining the length because it is of type: String[0...15]

**Comment by Bobbybigears** [comment]
Hi.

Many thanks for the prompt reply and clarification..

Cheers
Rob

On Mon, Sep 26, 2022 at 8:01 AM jpo-stx ***@***.***> wrote:

> Hello @Bobbybigears <https://github.com/Bobbybigears>
> the example is correct. The serial number needs to be prefixed with one
> byte defining the length because it is of type: String[0...15]
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Sevenstax/FreeV2G/discussions/84#discussioncomment-3731202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3HQFEJWNKI3QSNUNPS7BQTWAFC5PANCNFSM6AAAAAAQT4HCAI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Rob Davies
(Software Engineer)

Treharne Automotive Engineering Ltd
Treharne Workshops
Llanelli Gate
Dafen
Llanelli
Carmarthenshire
SA14 8LQ

Tel: 01554 775938
